src/_provider/metaOverviewAbstract.ts

```diff
@@ -109,17 +110,26 @@ export abstract class MetaOverviewAbstract {
   async init() {
     if (this.run) return this;
 
-    if (await this.getCache().hasValueAndIsNotEmpty()) {
+    const cache = await this.getCache();
```

Comment:
Does something speak against just using the locale inside of the cache key, which is defined at the end of this file?

Reply:
It does not. I just thought it would be logical to put locale inside value rather than key, want me to change?